changelog.txt

```diff
@@ -8,6 +8,10 @@ Added an overloaded version of NullnessUtil.castNonNull that takes an error mess
 
 Added a new option `-Aversion` to print the version of the Checker Framework.
 
+New CFGVisualizeLauncher command-line arguments:
+ * `-outdir`: directory in which to write output files
```

Comment:
The command-line arguments to this program not follow the convention that `-` is for single-character flags and `--` is for multi-character flags?  I think that would make the program easier to follow.

I realize this isn't your fault!  It could be fixed in this pull request or in a different pull request that is merged before this one.